src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskActionsController.java

```diff
@@ -206,18 +206,11 @@ public ResponseEntity<Void> completeTask(@RequestHeader(AUTHORIZATION) String au
         if (completeTaskRequest == null || completeTaskRequest.getCompletionOptions() == null) {
             taskManagementService.completeTask(taskId, accessControlResponse);
         } else {
-            boolean isPrivilegedRequest =
-                clientAccessControlService.hasPrivilegedAccess(serviceAuthToken, accessControlResponse);
-
-            if (isPrivilegedRequest) {
-                taskManagementService.completeTaskWithPrivilegeAndCompletionOptions(
-                    taskId,
-                    accessControlResponse,
-                    completeTaskRequest.getCompletionOptions()
-                );
-            } else {
-                throw new GenericForbiddenException(GENERIC_FORBIDDEN_ERROR);
-            }
+            taskManagementService.completeTaskWithPrivilegeAndCompletionOptions(
+                taskId,
+                accessControlResponse,
+                completeTaskRequest.getCompletionOptions()
+            );
```

Comment:
As per my point above, this will allow anyone to make a privileged request.  What we want is that _only_ ExUI can make these types of request.  We should continue to return a 403 when it's not.

Reply:
Methods added back. Only feature flag check removed.